### ISSUE TITLE
Docker image run with assume role option not working

### ISSUE THREAD

**Issue by cparadkar** [toniblyx/prowler#933]
docker run -ti --rm --name prowler --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --env AWS_SESSION_TOKEN toniblyx/prowler:latest "-A <accountid> -R <assumerolename>"
The above command is not wokring, gives following error:
" ERROR! - Both Account ID (-A) and IAM Role to assume (-R) must be set"

Please let us know if there is any alternate way of executing docker container run command with Assume role option.

**Comment by toniblyx** [comment]
Please read the advance usage section and assume role in the Readme. You need to add the command like this ' -A 123456789012 -R ProwlerRole'

**Comment by cparadkar** [comment]
I have added values with -A and -R still the error

**Comment by toniblyx** [comment]
join the discord server and we will try to help you there, I don´t understand what you trying to do and with what credentials... https://discord.gg/UjSMCVnxSB

**Comment by cparadkar** [comment]
When can we connect, I am located in India

**Comment by toniblyx** [comment]
see Discord we will try to help you out there

**Comment by cparadkar** [comment]
I have send the message on discord describing the issue, please look into it

**Comment by cparadkar** [comment]
Any update

**Comment by sectoramen** [comment]
Hi, this command works just fine for me. Please remove the quotes from the last part of the command and let me know if it works.

docker run -ti --rm --name prowler --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --env AWS_SESSION_TOKEN toniblyx/prowler:latest -A 1234567890 -R ProwlerExecRole

**Comment by cparadkar** [comment]
Similar command I also ran. The issue is that assessemnt runs but not for the account of the assumed role. It runs for the account which is configured in aws profile.

**Comment by toniblyx** [comment]
that was fixed already in de devel branch 2.7, can you try with:

```
docker run -ti --rm --name prowler \
--env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --env AWS_SESSION_TOKEN \
toniblyx/prowler:2.7-test -A 1234567890 -R ProwlerExecRole
```

**Comment by jfagoagas** [comment]
Hi @cparadkar, we are closing it. Please, check against branch 2.7, it's fixed there.

Also, feel free to reopen it if you experience this issue again.
